rate_based_rule/locals.tf

```diff
@@ -1,3 +1,10 @@
+
 locals {
-  tags = var.tags
-}
+  tags = {
+    "env"         = "dev"
+    "squad"       = "inf"
+    "product"     = "internal"
+    "type"        = "infra"
+  }
```

Comment:
Why did you add this? The tags should be passed into the module. Otherwise it will always be `squad = inf, env = dev` no matter who uses it